cli/src/commands/generate/util.ts

```diff
@@ -24,4 +25,12 @@ export function logRequiredMessage(logger: Logger, required: string[], instantia
     } else {
         logger.debug('Required properties: ' + required);
     }
+}
+
+export function appendPath(path: string[], element) : string[] {
```

Comment:
**minor** : You've typed everything here but element. Can I suggest you make this generic potentially?



```suggestion
export function appendPath<Type>(path: Type[], element: Type) : Type[] {
```

Then usage would be `appendPath<string>(array, element)`